docs/components/optimize/userguide/processes.md

```diff
@@ -20,5 +20,6 @@ Once a report is configured as KPI, its status can be seen on the **Processes Ov
 
 ## Configuring process owner and digests
 
-The process can be configured by clicking the 'Configure' button when hovering over the process. From this modal, you can change the owner of the process, as well as enable/disable the process digest. The process digest is a scheduled email report summarizing the current and previous state of the KPI reports for that process. It will be emailed to the owner of that process at the configured schedule. Please note that process digests are an alpha feature.
+The process can be configured by clicking the 'Configure' button when hovering over the process. From this modal, you can change the owner of the process, as well as enable/disable the process digest. The process digest is a scheduled email report summarizing the current and previous state of the KPI reports for that process. It will be emailed to the owner of that process at globally configurable regular intervals. Please note that process digests are an alpha feature.
```

Comment:
do you think it's worth linking to the config here? Or is this more user focused than technical?

Reply:
I wasn't sure if we want to avoid links that need maintained later on, but can't hurt